pkg/controllers/dynakube/connectioninfo/communication_hosts.go

```diff
@@ -5,19 +5,19 @@ import (
 	"strconv"
 	"strings"
 
-	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1/dynakube"
-	"github.com/Dynatrace/dynatrace-operator/src/dtclient"
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta1/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/clients/dynatrace"
```

Comment:
could `dtclient` alias be used here for `.../clients/dynatrace` import (as in many other places) ? The alias gives better context than `dynatrace`

Reply:
yeah agree `dtClient` alias is okey.